features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -1281,8 +1281,13 @@
   "console.identity_verification_providers.scopes.delete": ["internal_idvp_delete"],
   "console.insights.enabled": false,
   "console.login_and_registration.enabled": true,
+  "console.login_and_registration.disabled_features": ["ruleBasedPasswordExpiry"],
   "console.login_and_registration.scopes.feature": ["console:loginAndRegistration"],
-  "console.login_and_registration.scopes.read": ["internal_governance_view"],
+  "console.login_and_registration.scopes.read": [
```

Comment:
If a user is assigned an RBAC role that only has access to the loginAndRegistration feature, doesn’t this completely hide the loginAndRegistration section for them?

Reply:
Yes, that's correct! But there are a few things to consider:

- With the current behavior in Console settings, we only show the permissions for the main side panel feature.
- If we introduce a new feature under a main feature, the required permission should be included in the permissions list of the main feature in Console settings -> Roles.
- For example, if we take the Roles feature, then to display the Users section inside the Roles section, we must include the internal_user_mgt_list scope under the permissions of the Roles feature.
- IMO if we include these permissions under the main feature, then ideally the check should also be added under the main feature.

Reply:
The proper approach should be to implement a permission tree for these features, allowing them to be handled in a more granular way.

Reply:
Ack. Let's inform about the migration effort with above details.